### ISSUE TITLE
Strange characters/font in UI

### ISSUE THREAD

**Issue by justtobeme** [sparrowwallet/sparrow#1025]
Hi, 

I download your latest
![sparov](https://github.com/sparrowwallet/sparrow/assets/139632216/71c9c2b9-5aa9-40dd-9c04-8188fd4511d0)
 version of Sparrow Wallet and something strange is happen. Can you please let me know how to fix this issue?

Kindly looking forward to hearing from you. 

Sincerely,

Bojan

**Comment by craigraw** [comment]
Try downloading and reinstalling the Segoe UI font from https://freefontsfamily.com/segoe-ui-font-free/

You will need to double click on the SEGOEUI.TTF file in that download after you have unzipped it.

**Comment by justtobeme** [comment]
Tnx, it is working now.

**Comment by bitcoinRph** [comment]
<img width="1277" alt="Screen Shot 2025-01-07 at 9 10 39 PM" src="https://github.com/user-attachments/assets/63dc84af-4fa5-420c-8cc9-2c773aedcaad" />

I downloaded version 1.9.1 of sparrow wallet and this is what shows up. I completed the above steps to download the font and made sure it was added into my active fonts. Deleted and re-downloaded sparrow including restart of hardware and this still shows up for me. 

I am running MacOS Sierra Version 10.12.6

Thanks
